packages/Popover/src/Popover.js

```diff
@@ -455,10 +461,13 @@ class Popover extends React.Component {
 
 // Todo Refactor this when we convert popover component to use hooks
 function PopoverChildren(props) {
+  const { children, onChildChange } = props;
+
   React.useLayoutEffect(() => {
-    if (props.children) props.onChildChange();
-  }, [props.children]);
-  return props.children;
+    if (children) onChildChange();
+  }, [children, onChildChange]);
```

Comment:
@mikrotron Can you explain why you needed to add the `onChildChange` prop here? Is that so it detects if that prop is dynamically added?

Reply:
In _theory_ that prop could change, and this should react on that.  Really though, it was just to make eslint happier (it was warning about it).